spec/require_encapsulation_check.rb

```diff
@@ -0,0 +1,31 @@
+require 'English'
+REQUIRES = %w[
+  datadog/appsec
+  datadog/core
+  datadog/kit
+  datadog/profiling
+  datadog/tracing
+].freeze
```

Comment:
This constant will be leaking globally, as is it should probably go into the `RSpec.describe` block.

Could even be a plain local variable, or even just `%w[ ... ].each do |req|`.